test/server/index.test.ts

```diff
@@ -1,12 +1,81 @@
-import { C } from "./../../src/server";
+import request from "supertest";
+import { app } from "../../src/server/index";
 
-describe("Sample Test", function () {
-  it("should pass harmlessly", function () {
-    const c = new C();
+describe("App Server Tests", function () {
+  describe("Routes", function () {
+    it("should reject falsy routes and return 404 not found", function (done) {
+      request(app)
```

Comment:
**Comment**: Please rename `request` binding to `supertest`. There already exists a package named `request`, and naming it this way is unintuitive.

Reply:
Renamed the import to supertest for clarification.